### DISCUSSION TITLE
Output or input system audio to different channels (Mac)

### DISCUSSION THREAD

**Discussion by stereokai** [ExistentialAudio/BlackHole#703] [Q&A]
I understand that by default, the system audio is routed to channels 1 and 2. But is it possible to set the default system output device to any other pair of channels instead?

Thanks

PS. This piece of software is rocking!

**Comment by devinroth** [comment]
No easily.

What’s your end goal here?

Devin

**Reply by stereokai** [reply]
Hi Devin, I'm sorry, I missed your reply. Basically, I'd like more flexibility when routing audio between sources, DAW and audio interface

**Comment by devinroth** [comment]
Not sure how this would help. You are able to do all sorts of complex routing inside a DAW. What benefit would this add? I’m just not seeing the vision.

Devin
